test/test_nestedtensor.py

```diff
@@ -7165,13 +7172,17 @@ def f(*args, **kwargs):
             out_ref = f(sample.input, *sample.args, **sample.kwargs)
             out_compile = compiled_f(sample.input, *sample.args, **sample.kwargs)
 
-            self.assertEqual(out_compile, out_ref)
+            if op.full_name in COMPARE_TENSOR_COMPONENT_EQUALITY:
+                self.assertEqual(out_compile.unbind(), out_ref.unbind())
```

Comment:
nit: we have a utility that does exactly this (except also supports arbitrary number of outputs)
`self.assertEqualIgnoringNestedInts` that can be swapped in place of `self.assertEqual`

Reply:
thanks, was looking for this!